doc/source/overview.rst

```diff
@@ -24,20 +28,19 @@ Package overview
  * Static and moving window linear and `panel regression
    <http://en.wikipedia.org/wiki/Panel_data>`__
 
-Data structures at a glance
----------------------------
+Data Structures
+---------------
 
 .. csv-table::
     :header: "Dimensions", "Name", "Description"
     :widths: 15, 20, 50
 
-    1, Series, "1D labeled homogeneously-typed array"
-    2, DataFrame, "General 2D labeled, size-mutable tabular structure with
-    potentially heterogeneously-typed columns"
-    3, Panel, "General 3D labeled, also size-mutable array"
+    1, "Series", "1D labeled homogeneously-typed array"
+    2, "DataFrame", "General 2D labeled, size-mutable tabular structure with potentially heterogeneously-typed column"
+    3, "Panel", "General 3Dlabeled, also size-mutable array"
```

Comment:
Should we leave out Panel ? 
(since it is deprecated we shouldn't publicize it I think) 
If we keep, should be a space between '3D' and 'labeled'